### ISSUE TITLE
AWS cdk deploy and profile regions

### ISSUE THREAD

**Issue by RichardScottOZ** [developmentseed/titiler-pds#4]
This could be a discussion, but thought I would note:

I was using Windows 10 and the CDK installed last week 1.86.

Using cdk deploy --profile   tilerprof
for example

correctly read the keys for permissions for deployment - but failed to get the profile from the config, however I tried it

Not a titiler problem, but a cdk problem.  Suggestions on their Slack were 'put the right environment variables in your code' - which I said 'not my code'.

Thought I would add, just in case anyone else comes across.

Basically, I changed my AWS default region environment variable, deployed, set it back.  e.g. mine is ap-southeast-2 as that is where we are usually working.  Changed it to us-west-2 for the Sentinel COGs and set it back.

Only thing that caused me any problems except I think changing python3 to python in the app to get it to work for the environment I was using.
